cocos/serialization/deserialize-dynamic.ts

```diff
@@ -235,7 +235,7 @@ function compileDeserializeNative (_self: _Deserializer, klass: CCClassConstruct
             if (prop === undefined) {
                 continue;
             }
-            if (!fastMode && typeof prop !== 'object') {
+            if (typeof prop !== 'object') {
                 o[propName] = prop;
```

Comment:
There is no problem with this change. fastMode was judged only because it was assumed that no type serialized fields would be officially declared. In fact, this assumption should not affect the actual code logic.

Reply:
Solved after conversation.